sbi/inference/snpe/snpe_base.py

```diff
@@ -137,6 +137,20 @@ def append_simulations(
         self._x_roundwise.append(x)
         self._proposal_roundwise.append(proposal)
 
+        if self._prior is None or isinstance(self._prior, EmpiricalImproper):
+            if proposal is not None:
+                raise ValueError(
+                    "You had not passed a prior at initialization, but now you "
```

Comment:
I would prefer passive form. 
"A proposal was passed but no prior was passed at initialisation. When running multi-round inference, a prior needs to be specified upon initialisation. Potential fix: setting the `.prior` attribute or re-initialisation. If the sampled passed to `append_simulations()` were sampled from the prior, single-round inference can be performed with `append_simulations(..., proprosal=None)`."